presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -124,12 +199,62 @@ protected ResultSet getTables(Connection connection, String schemaName, String t
         return super.toPrestoType(session, typeHandle);
     }
 
+    @Override
+    protected Optional<JdbcTypeHandle> getArrayElementTypeHandle(JdbcTypeHandle arrayTypeHandle)
+    {
+        try (Connection connection = connectionFactory.openConnection()) {
+            // PostgreSql array typenames are the base element type prepended with "_"
+            String elementPgTypeName = arrayTypeHandle.getJdbcTypeName().replaceAll("_", "");
```

Comment:
I feel `.substring(1)` is better than `replaceAll` if you want to just remove first character `_`.